app.go

```diff
@@ -693,7 +756,11 @@ func (app *App) executeInvokes() error {
 		}
 
 		if err != nil {
-			app.logger.Printf("fx.Invoke(%v) called from:\n%+vFailed: %v", fname, i.Stack, err)
+			fxlog.Error(
+				"fx.Invoke failed",
+				fxlog.F("called from", fname),
```

Comment:
calledFrom

Reply:
A message would then look like `fx.Invoke failed calledFrom` which is a bit awkward, no?

Reply:
No, because the structured logs will be written as JSON like:

```
{"msg": "fx.Invoke failed", "calledFrom": "..."}
```